ui/src/app/components/invalid-billing-banner.spec.tsx

```diff
@@ -0,0 +1,198 @@
+import '@testing-library/jest-dom';
```

Comment:
this file is not in the same location as what it's testing.  Please move one or the other.